snowplow-tracker/src/main/java/com/snowplowanalytics/snowplow/tracker/Tracker.java

```diff
@@ -888,7 +893,9 @@ public ScreenState getScreenState() {
 
     /**
      * Track a screen with a screen state (many options)
+     * @deprecated Use track method passing a ScreenView event.
```

Comment:
I think might be a little clearer as: Use track(Event) method passing a ScreenView event

I only suggest this as I went looking for trackScreen(ScreenView) first.